en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect-with-virtual-hosts.md

```diff
@@ -0,0 +1,138 @@
+# Deploy API to Choreo Connect with Virtual Hosts 
+
+There are two ways to add an API to the Choreo Connect.
```

Comment:
```suggestion
There are two ways to add an API to Choreo Connect.
```
Can we also add why we want to do this so that new users get a full picture when they land on this page?

Reply:
fixed with https://github.com/wso2/docs-apim/commit/b9900b303fe9c36a8933801e2dd3f8b605cab370 and add a reference to the overview page if the user wants more info.